plugins/app-backend/src/service/router.ts

```diff
@@ -95,9 +95,28 @@ export async function createRouter(
   staticRouter.use(notFoundHandler());
 
   router.use('/static', staticRouter);
-  router.use(express.static(appDistDir));
+  router.use(
+    express.static(appDistDir, {
+      setHeaders: (res, path) => {
+        // The Cache-Control header instructs the browser to not cache html files since it might
+        // link to static assets from recently deployed versions. This is a workaround when no
```

Comment:
There's still dynamic loading to take into account, and I'm thinking it's time to build out the docs for that and approaches for implementing the `staticFallbackHandler` soon